packages/cfd/src/Containers/jurisdiction-modal.tsx

```diff
@@ -92,13 +100,26 @@ const JurisdictionModal = ({
     const poa_failed = poa_status === 'suspected' || poa_status === 'rejected' || poa_status === 'expired';
     const poi_poa_not_submitted = poi_status === 'none' || poa_status === 'none';
 
-    React.useEffect(() => {
-        if (is_jurisdiction_modal_visible) {
-            if ((poa_status === 'pending' || poi_status === 'pending') && !is_eu) {
+    const selectSVGJurisdiction = () => {
+        if (account_type.type && account_type.category && !is_eu) {
```

Comment:
@amina-deriv could you please check if this condition will work the same way without account_type.category on this line? 🙂
```suggestion
        if (account_type.type && !is_eu) {
```

Reply:
Removed the extra condition. Thanks @maryia-binary